src/type-coercions.md

```diff
@@ -195,10 +195,14 @@ r[coerce.unsize]
 
 r[coerce.unsize.intro]
 The following coercions are called `unsized coercions`, since they
-relate to converting sized types to unsized types, and are permitted in a few
+relate to converting types to unsized types, and are permitted in a few
 cases where other coercions are not, as described above. They can still happen
 anywhere else a coercion can occur.
 
+r[coerce.unsize.misnomer]
+> Note: "unsizing" is a bit of a misnomer,
+> since this covers unsized->unsized coercions too.
```

Comment:
```suggestion
> Note: These are sometimes called "unsizing" coercions, but that can be a bit of a misnomer, since these cover unsized->unsized coercions too.
```

Not sure if something like this is what you meant to say or not.  The Reference doesn't refer to "unsizing" anywhere -- even though, you're right, we often do in conversation -- so we'd need to introduce that somehow in this note.

Reply:
Hm. There may not be the literal word "unsizing", but we do refer to this coercion as "unsize coercion". 

> The following coercions are called `unsized coercions`, since they

I do feel like it's clear what this means, given that this note is surrounded by mentions of unsize coercions. (but maybe not see you seem to be confused?)

Reply:
Although I'd also say that replacing "unsized coercion" to "unsizing coercion" everywhere in the reference would be a good idea. It sounds better and more closely matches how people actually talk about this feature.

Reply:
In conversation with @traviscross, I was one of the confused ones. TC pointed out that "unsized coercion" can be interpreted as the codomain so to speak. That is, it doesn't seem like a misnomer to me because it is only referring to coercions *to* something not sized, and thus seemed accurate.

It is true the word "unsizing" implies "to remove the size of", that is the *domain* is sized, and thus is not always an accurate term. However, we don't use "unsizing" here.

Probably a bit of a muddled interpretation on my part, but does that make sense?

Reply:
"unsized coercion" sounds like the coercion itself is unsized. I don't even know what that should mean.